### PR TITLE
refactor(deepl): cleanup

### PR DESCRIPTION
**Pull request by somerand0mcat**
## 🔧 What does this fix? 🔧
as said in the title

## 🗒 Checklist 🗒

- [X] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [X] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

**Review by isabelroses — APPROVED** [review]

**Review by uncenter — CHANGES_REQUESTED** [review]
You need to remove the `& when (@lookup = latte) {` parts now.

**Review by uncenter — CHANGES_REQUESTED** [review]

File: styles/deepl/catppuccin.user.css [hunk 1]
```diff
@@ -293,89 +295,28 @@
...
-        filter: brightness(0) invert(86%) sepia(4%) saturate(1588%)
-          hue-rotate(192deg) brightness(101%) contrast(91%);
-      }
-
-      .Logo-module--logoImg--1ee7e,
-      .deeplLogo-module--logo--ffd7a {
-        filter: invert(61%) sepia(85%) saturate(375%) hue-rotate(189deg)
-          brightness(98%) contrast(100%);
-      }
+    .Logo-module--logoImg--1ee7e {
```
[the remark this answers is not in the document]
Comment (somerand0mcat) [reply styles/deepl/catppuccin.user.css:308]: i think i missed that element yeah

File: styles/deepl/catppuccin.user.css [hunk 2]
```diff
@@ -293,89 +295,29 @@
...
+  @mocha:     { @rosewater: #f5e0dc; @flamingo: #f2cdcd; @pink: #f5c2e7; @mauve: #cba6f7; @red: #f38ba8; @maroon: #eba0ac; @peach: #fab387; @yellow: #f9e2af; @blue_filter: invert(61%) sepia(85%) saturate(375%) hue-rotate(189deg) brightness(98%) contrast(100%); @green: #a6e3a1; @teal: #94e2d5; @sky: #89dceb; @sapphire: #74c7ec; @blue: #89b4fa; @lavender: #b4befe; @text: #cdd6f4; @text_filter: brightness(0) saturate(100%) invert(87%) sepia(6%) saturate(985%) hue-rotate(191deg) brightness(96%) contrast(99%); @subtext1: #bac2de; @subtext0: #a6adc8; @overlay2: #9399b2; @overlay1: #7f849c; @overlay0: #6c7086; @surface2: #585b70; @surface1: #45475a; @surface0: #313244; @base: #1e1e2e; @mantle: #181825; @crust: #11111b; };
```
Comment (uncenter) [inline styles/deepl/catppuccin.user.css:320]: Why are all the filters the exact same lol. Defeats the point of this whole thing. Use https://angel-rs.github.io/css-color-filter-generator/ on the color for each flavor, and keep clicking until it gets less then 0.3 precision.

**Review by uncenter — CHANGES_REQUESTED** [review]

File: styles/deepl/catppuccin.user.css (hunk 2 above)
Comment (somerand0mcat) [reply styles/deepl/catppuccin.user.css:320]: > Why are all the filters the exact same lol. Defeats the point of this whole thing. Use https://angel-rs.github.io/css-color-filter-generator/ on the color for each flavor, and keep clicking until it gets less then 0.3 precision.

wait i think i fucked up during copy and pasting the filters

Comment (somerand0mcat) [reply styles/deepl/catppuccin.user.css]: anyways though, i fixed the filters

Comment (uncenter) [reply styles/deepl/catppuccin.user.css]: They don't look to be accurate. You can use the following JS snippet on the filter website to click the button for you until it is accurate.

```js
while (Number.parseFloat(document.querySelector('#lossDetail').firstChild.textContent.split(':')[1].trim().slice(0, -1)) !== 0.1) {
  compute()
  console.log('Computed value.')
}
```

Comment (somerand0mcat) [reply styles/deepl/catppuccin.user.css]: i think its accurate now

**Review by watatomo — APPROVED** [review]

**Comment by uncenter** [comment]
Please stop updating this branch Omar. It sends a notification to everyone here, it isn't required for merging, and nothing about the CI has changed since you last updated.

**Comment by somerand0mcat** [comment]
> Please stop updating this branch Omar. It sends a notification to everyone here, it isn't required for merging, and nothing about the CI has changed since you last updated.

wait every single commit i do sends a notification?

**Comment by uncenter** [comment]
> wait every single commit i do on a PR sends a notification?

...yep (technically every push but yeah).

**Comment by somerand0mcat** [comment]
> ...yep (technically every push but yeah).
well i guess i'll just stop

**Review by uncenter — APPROVED** [review]

**Review by watatomo — APPROVED** [review]

**Pull request merged**